src/Tasks.kt

```diff
@@ -0,0 +1,10 @@
+package test.agileFreaks
+
+fun IntArray.sumOfEven():Int {
+    var sum =0
+    this.forEach {
+        if (it % 2 == 0) sum += it
```

Comment:
If you really want to refactor, how about an extension method to encapsulate the even logic?

Reply:
Awesome idea, thanks for the info!